<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiCriteriaEvaluationWidgetBase</class>
 <widget class="QWidget" name="MultiCriteriaEvaluationWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="MolusceTableWidget" name="tblMatrix"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="MolusceTableWidget" name="tblWeights"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From class</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spnInitialClass"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To class</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spnFinalClass"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnTrainModel">
     <property name="text">
      <string>Train model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pairwise Comparison Matrix</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Weights Matrix</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MolusceTableWidget</class>
   <extends>QTableWidget</extends>
   <header>molusce.moluscetablewidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
